<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>history_dialog</class>
 <widget class="QDialog" name="history_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>历史记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_history1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_history1_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_history1_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_history1_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_history1_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_history1_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_upgrad">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
